CompetitionTask/Pages/EducationFeature.cs

```diff
@@ -0,0 +1,265 @@
+﻿using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using NUnit.Framework;
+using System.Diagnostics.Metrics;
+namespace Mars_Education_Certifications.Pages
+{
+
+    public class EducationFeature
```

Comment:
also this should be named Education page

Reply:
Resolved